node/src/Consumer.ts

```diff
@@ -1104,27 +1141,27 @@ function parseSimpleConsumerDump(data: FbsConsumer.SimpleConsumerDump): SimpleCo
 }
 
 function parseSimulcastConsumerDump(
-	data: FbsConsumer.SimulcastConsumerDump
+	data: FbsConsumer.ConsumerDump
 ) : SimulcastConsumerDump
 {
 	const base = parseBaseConsumerDump(data.base()!);
-	const rtpStream = parseRtpStream(data.rtpStream()!);
+	const rtpStream = parseRtpStream(data.rtpStreams(0)!);
 
 	return {
 		...base,
 		type                   : 'simulcast',
 		rtpStream,
-		preferredSpatialLayer  : data.preferredSpatialLayer(),
-		targetSpatialLayer     : data.targetSpatialLayer(),
-		currentSpatialLayer    : data.currentSpatialLayer(),
-		preferredTemporalLayer : data.preferredTemporalLayer(),
-		targetTemporalLayer    : data.targetTemporalLayer(),
-		currentTemporalLayer   : data.currentTemporalLayer()
+		preferredSpatialLayer  : data.preferredSpatialLayer()!,
+		targetSpatialLayer     : data.targetSpatialLayer()!,
+		currentSpatialLayer    : data.currentSpatialLayer()!,
+		preferredTemporalLayer : data.preferredTemporalLayer()!,
+		targetTemporalLayer    : data.targetTemporalLayer()!,
+		currentTemporalLayer   : data.currentTemporalLayer()!
```

Comment:
The number of `!` looks worrying to me, something feels off if we need it so much in this function

Reply:
For simulcast and SVC we know these have values coming from the worker, this is the reason of using `!` here. For `SimpleConsumer` they are not being populated and we don't even access them.